### PR TITLE
chore: add template files for integration instructions

### PR DESCRIPTION
**Pull request by MrTibbles**
## Changes
- Adds template files for clear instructions when integrating
- Adds to supported css variables for typography

## Notes
The confluence document will be tweaked and amended but the thinking is to have a single source of truth for documentation, that being Confluence

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,41 +1,3 @@
...
-  operator: 'img',
-  sport: 'golf',
-  eventId: '264',
-  targetElementSelector: '#event-centre-dom-element',
-  targetModule: 'full',
-  version: 'latest',
-  theme: 'img-dark-preview'
-});
-```
+Please see the [confluence document for instructions](https://imgarenahub.atlassian.net/wiki/spaces/IAT/pages/1564082192/Operator+Style+Overrides).
```
Comment (parkjh) [inline README.md:3]: any reason why you chose confluence as the source of truth?

Comment (parkjh) [reply README.md:3]: as far as I can tell, it doesn't seem like anyone else beside devs need to know about this?

**Comment by parkjh** [comment]
one question but lgtm

File: README.md (hunk 1 above)
Comment (MrTibbles) [reply README.md:3]: We (tech teams in general) are trying to centralise as much documentation as possible on confluence. Also, this process of operator style overrides will be carried out by the integrations team, not developers, who are more accustomed to using Confluence.

**Review by parkjh — APPROVED** [review]

**Pull request merged**